src/Actions/ManagesStatuses.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace TestMonitor\Clickup\Actions;
+
+use TestMonitor\Clickup\Transforms\TransformsStatuses;
+
+trait ManagesStatuses
+{
+    use TransformsStatuses;
+
+    /**
+     * Get a list of available statuses for the provided space.
+     *
+     * @throws \TestMonitor\Clickup\Exceptions\InvalidDataException
+     *
+     * @return \TestMonitor\Clickup\Resources\Status[]
+     */
+    public function statuses($spaceId)
```

Comment:
The parameter `$spaceId` is missing from the doc blocks.

Reply:
Fixed